Cargo.toml

```diff
@@ -38,11 +38,11 @@ prover-logger = { path = "crates/prover-logger" }
 prover-utils = { path = "crates/prover-utils" }
 
 agglayer-telemetry = { git = "https://github.com/agglayer/agglayer.git", branch = "release/0.2.1" }
-agglayer-primitives = { git = "https://github.com/agglayer/interop.git", tag = "v0.1.0" }
-agglayer-interop = { git = "https://github.com/agglayer/interop.git", tag = "v0.1.0" }
+agglayer-primitives = { git = "https://github.com/agglayer/interop.git", rev = "e38287a" }
+agglayer-interop = { git = "https://github.com/agglayer/interop.git", rev = "e38287a" }
 
```

Comment:
I'll make a proper tag once we're good, just in case we require some changes in interop during review

Reply:
> I'll make a proper tag once we're good, just in case we require some changes in interop during review

If the changes you need are included in main of the interop repo already, I think you can just remove the `rev` key and use `Cargo.lock` to nail down a specific version for development.